File: cpp/Program/Car.cpp
#pragma once
#include <iostream>
#include <string>
#include "Vehicle.cpp"

using namespace std;

class Car : public Vehicle
{
private:
    int jumKursi;
    int jumPintu;
    

public:
    Car(){

    }

    Car(int jumKursi, string platNomor, string merk, string tahunProduksi, string warna) : Vehicle(platNomor, merk, tahunProduksi, warna){
        this->jumKursi = jumKursi;
        jumPintu = 0;
    }

    Car(int jumKursi, int jumPintu, string platNomor, string merk, string tahunProduksi, string warna) : Vehicle(platNomor, merk, tahunProduksi, warna){
        this->jumKursi = jumKursi;
        this->jumPintu = jumPintu;
    }

    int GetJumKursi()  {
        return jumKursi;
    }

    void SetJumKursi(int jumKursi) {
        this->jumKursi = jumKursi;
    }

    int GetJumPintu()  {
        return jumPintu;
    }

    void SetJumPintu(int jumPintu) {
        this->jumPintu = jumPintu;
    }

    ~Car(){

    }
};

File: python/Program/Car.py
from Vehicle import Vehicle
class Car(Vehicle):
    __jumKursi = ''
    __jumPintu = ''

    def __init__(self, platNomor = "", merk = "", tahunProduksi = "", warna = "", jumKursi = '', jumPintu = ''):
        super().__init__(platNomor, merk , tahunProduksi , warna )
        self.__jumKursi = jumKursi
        self.__jumPintu = jumPintu

    def get_jumKursi(self):
        return self.__jumKursi

    def set_jumKursi(self, value):
        self.__jumKursi = value

    def get_jumPintu(self):
        return self.__jumPintu

    def set_jumPintu(self, value):
        self.__jumPintu = value

        
    